README.md

```diff
@@ -117,7 +117,7 @@ Yes, ESLint natively supports parsing JSX syntax (this must be enabled in [confi
 
 ### What ECMAScript versions does ESLint support?
 
-ESLint has full support for ECMAScript 3, 5 (default), 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, and 2023. You can set your desired ECMAScript syntax (and other settings, like global variables or your target environments) through [configuration](https://eslint.org/docs/latest/use/configure).
+ESLint has full support for ECMAScript 3, 5, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, and 2023. ECMAScript 2024 (the default) is also supported, although the specification hasn't been finalized yet. You can set your desired ECMAScript syntax and other settings (like global variables) through [configuration](https://eslint.org/docs/latest/use/configure).
```

Comment:
I think it's time to make this a lot more generic instead of needing to update it every year. Maybe something like:

```suggestion
ESLint has full support for ECMAScript 3, 5, and every year up until the most current stage 4 specification. You can set your desired ECMAScript syntax and other settings (like global variables) through [configuration](https://eslint.org/docs/latest/use/configure).
```

Reply:
It's also easier to read 👍🏻